toolkit/committee-selection/authority-selection-inherents/src/filter_invalid_candidates.rs

```diff
@@ -104,7 +121,7 @@ where
 		.collect()
 }
 
-pub fn filter_invalid_permissioned_candidates<TAccountId, TAccountKeys>(
+pub(crate) fn filter_invalid_permissioned_candidates<TAccountId, TAccountKeys>(
```

Comment:
I would leave it a public function in the toolkit and not require users to re-implement it if they would like to have their own implementation of committee selection.

It is well defined function that filters entries containing invalid keys from input.